.github/workflows/publish.yaml

```diff
@@ -78,7 +78,7 @@ jobs:
           echo "Tagging with ${{ inputs.image-tag }}"
           echo ${{ secrets.GITHUB_TOKEN }} | docker login $IMAGE_REGISTRY -u $GITHUB_ACTOR --password-stdin
           docker pull $API_IMAGE
-          printf "$(git log -n 1 --format=format:'%h% %>(12) %cs %aN%  %s %d')" > ./api/src/version.txt
+          printf "$(git log -n 1 --format=format:'%h% %>(12) %cs %aN%  "%s"%d' --decorate=short --decorate-refs=refs/tags)" > ./api/src/version.txt
```

Comment:
Why do you want to store the git hash, date, commit message and author when you don't use them?

Reply:
Don't really have a better answer than "that's how the template does it". @sooftsad can probably elaborate.